usaspending_api/api_docs/api_documentation/spending_explorer.md

```diff
@@ -0,0 +1,108 @@
+## [Spending Explorer](#spending-explorer)
+
+**Route:** `/api/v2/spending/`
+
+**Method:** `POST`
+
+This route sends a request to the backend to retrieve spending data information through various types and filters.
+
+### Request
+
+```
+{
+    "type": "agency",
+    "filters": {
+        "fy": "2017"
+    }
+}
+```
+**Query Parameters Description**
+
+**type** - `required` - a string that contains the type of spending to explore on. It must be one of these values: *budget_function*, *budget_subfunction*, *federal_account*, *program_activity*, *object_class*, *recipients*, *awards*, *agency*
+
+**filters** - `optional` - string index values to filter explorers on.  The *quarter* field is cumulative and quarters are only available 45 days after their close. Also quarter data is not available before FY 2017 Q2.
```

Comment:
`filters` is an object.

**filters** - `optional` - an object with the following key-value pairs that indicate what subset of results will be displayed. All values should be strings.

Reply:
added ur comment but kept the quarter content